src/views/AllThreeWActivity/index.tsx

```diff
@@ -0,0 +1,197 @@
+type ProjectListItem = NonNullable<ProjectsResponse['results']>[number];
+type ActivityListItem = NonNullable<ProjectListItem['activities']>[number];
+type DistrictListItem = NonNullable<ProjectListItem['districts_details']>[number];
+
+const ITEM_PER_PAGE = 15;
+type TableKey = number;
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const strings = useTranslation(i18n);
+    const [projectActivePage, setProjectActivePage] = useState(1);
+    const {
+        response: projectResponse,
+        pending: projectResponsePending,
+    } = useRequest({
+        url: '/api/v2/emergency-project/',
+        preserveResponse: true,
+        query: {
+            limit: ITEM_PER_PAGE,
+            offset: ITEM_PER_PAGE * (projectActivePage - 1),
+        },
+    });
+
+    function getPeopleReachedInActivity(activity: ActivityListItem) {
+        const {
+            is_simplified_report,
+            people_count,
+            male_0_1_count,
+            male_2_5_count,
+            male_6_12_count,
+            male_13_17_count,
+            male_18_59_count,
+            male_60_plus_count,
+            male_unknown_age_count,
+            female_0_1_count,
+            female_2_5_count,
+            female_6_12_count,
+            female_13_17_count,
+            female_18_59_count,
+            female_60_plus_count,
+            female_unknown_age_count,
+            other_0_1_count,
+            other_2_5_count,
+            other_6_12_count,
+            other_13_17_count,
+            other_18_59_count,
+            other_60_plus_count,
+            other_unknown_age_count,
+        } = activity;
+
+        if (is_simplified_report === true) {
+            return people_count ?? 0;
+        }
+
+        if (is_simplified_report === false) {
+            return sumSafe([
+                male_0_1_count,
+                male_2_5_count,
+                male_6_12_count,
+                male_13_17_count,
+                male_18_59_count,
+                male_60_plus_count,
+                male_unknown_age_count,
+
+                female_0_1_count,
+                female_2_5_count,
+                female_6_12_count,
+                female_13_17_count,
+                female_18_59_count,
+                female_60_plus_count,
+                female_unknown_age_count,
+
+                other_0_1_count,
+                other_2_5_count,
+                other_6_12_count,
+                other_13_17_count,
+                other_18_59_count,
+                other_60_plus_count,
+                other_unknown_age_count,
+            ]);
+        }
+        return undefined;
+    }
+
+    function DistrictNameOutput({ districtName }: { districtName: string }) {
+        return districtName;
+    }
+
+    const getPeopleReached = useCallback(
+        (project: ProjectListItem) => sumSafe(project.activities?.map(getPeopleReachedInActivity)),
+        [],
+    );
```

Comment:
Let's move all of these function outside the component as these do not depend on any of the context inside the function